[2025-05-13T07:47:55.933+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-13T07:47:55.970+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sec_filing_scheduler.process_alny manual__2025-05-13T07:47:38.690246+00:00 [queued]>
[2025-05-13T07:47:55.981+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sec_filing_scheduler.process_alny manual__2025-05-13T07:47:38.690246+00:00 [queued]>
[2025-05-13T07:47:55.983+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-05-13T07:47:56.004+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): process_alny> on 2025-05-13 07:47:38.690246+00:00
[2025-05-13T07:47:56.044+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'sec_filing_scheduler', 'process_alny', 'manual__2025-05-13T07:47:38.690246+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/orchestrate_clinical_pipeline.py', '--cfg-path', '/tmp/tmptq9j9qic']
[2025-05-13T07:47:56.056+0000] {standard_task_runner.py:105} INFO - Job 40: Subtask process_alny
[2025-05-13T07:47:56.045+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=647) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-13T07:47:56.058+0000] {standard_task_runner.py:72} INFO - Started process 651 to run task
[2025-05-13T07:47:56.259+0000] {task_command.py:467} INFO - Running <TaskInstance: sec_filing_scheduler.process_alny manual__2025-05-13T07:47:38.690246+00:00 [running]> on host ae64a678dd42
[2025-05-13T07:47:56.503+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sec_filing_scheduler' AIRFLOW_CTX_TASK_ID='process_alny' AIRFLOW_CTX_EXECUTION_DATE='2025-05-13T07:47:38.690246+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-13T07:47:38.690246+00:00'
[2025-05-13T07:47:56.504+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-13T07:47:56.521+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-05-13T07:47:56.523+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-05-13T07:47:58.332+0000] {logging_mixin.py:190} INFO - Found 141 8-K filings
[2025-05-13T07:47:58.334+0000] {logging_mixin.py:190} INFO - Found 13 10-K filings
[2025-05-13T07:47:58.335+0000] {logging_mixin.py:190} INFO - Found 37 10-Q filings
[2025-05-13T07:47:58.337+0000] {logging_mixin.py:190} INFO - Found 191 new filings.
[2025-05-13T07:47:58.338+0000] {logging_mixin.py:190} INFO - Processing 1 filings...
[2025-05-13T07:48:00.398+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-05-13T07:48:00.403+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-05-13T07:48:00.405+0000] {logging_mixin.py:190} INFO - Finished 1/1: 0001178670-25-000052
[2025-05-13T07:48:00.406+0000] {logging_mixin.py:190} INFO - check 1
[2025-05-13T07:48:00.409+0000] {logging_mixin.py:190} INFO - check 2
[2025-05-13T07:48:00.411+0000] {logging_mixin.py:190} INFO - check 3
[2025-05-13T07:48:00.429+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-05-13T07:48:00.431+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-05-13T07:48:00.433+0000] {logging_mixin.py:190} INFO - Error processing filings: the JSON object must be str, bytes or bytearray, not NoneType
[2025-05-13T07:48:00.437+0000] {logging_mixin.py:190} INFO - Deleted output_dir: sec_data/ALNY
[2025-05-13T07:48:00.438+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-13T07:48:00.470+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-13T07:48:00.473+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=sec_filing_scheduler, task_id=process_alny, run_id=manual__2025-05-13T07:47:38.690246+00:00, execution_date=20250513T074738, start_date=20250513T074755, end_date=20250513T074800
[2025-05-13T07:48:00.664+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-13T07:48:00.712+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-13T07:48:00.726+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
